apps/web/frontend/components/workHourCommentButton.tsx

```diff
@@ -11,7 +11,7 @@ export const WorkHourCommentFragment = graphql(`
   fragment WorkHourCommentFragment on Task {
     id
     title
-    workHourOfDays(from: $from, to: $to) {
+    workHourOfDays(from: $from, to: $to, forUserId: $forUserId) {
```

Comment:
I think it doesn't do anything here and a few more places below
```suggestion
    workHourOfDays(from: $from, to: $to) {
```

Reply:
I receive this error if those are not explicitly added:
```
GraphQL Document Validation failed with 4 errors [FAILED] 
Error 0: Fields "tasks" conflict because subfields "workHourOfDays" conflict because they have differing arguments. Use different aliases on the fields to fetch both if this was intentional.
```